app/audit/models/WhatYouOweResponseAuditModel.scala

```diff
@@ -66,29 +66,35 @@ case class WhatYouOweResponseAuditModel(user: MtdItUser[_],
     else Json.obj()
   }
 
-  private def remainingToPay(documentDetail: DocumentDetail): BigDecimal = {
-    if (documentDetail.isLatePaymentInterest) documentDetail.interestRemainingToPay else documentDetail.remainingToPay
-  }
+  def getChargeType(chargeItem: ChargeItem, latePaymentCharge: Boolean): Option[String] =
```

Comment:
this looks like similar to getChargeType in TaxYearSummaryResponseAuditModel, worth to create a base trait for this shared function I think as this stage?